tests/fixtures/services.py

```diff
@@ -33,7 +34,11 @@ def mock_services_container(
         container._container_instance = None
 
 
-class MockServicesFixture:
+class ServicesFixture:
```

Comment:
This is where the bulk of the logic changes in this PR are. Most of the other changes are changes to tests to remove all the wiring we were doing.